Create accessibility-how-to.md

hhartingrex opened the pull request:
@mgifford Here is a starting place. Once we get this page working - first draft - I will adjust the left navigation bar. Thank you!

<!-- readthedocs-preview civicactions-handbook start -->
----
:books: Documentation preview :books:: https://civicactions-handbook--1086.org.readthedocs.build/en/1086/

<!-- readthedocs-preview civicactions-handbook end -->

hhartingrex committed:
Create accessibility-how-to.md

@mgifford Here is a starting place. Once we get this page working - first draft - I will adjust the left navigation bar. Thank you!

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hhartingrex committed:
Update mkdocs.yml

Adding in left nav changes
NOTE: Hoping I got the two changes into one PR!

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

dmundra reviewed (commented):
@hhartingrex do you want to deploy a work in progress page or hold off till it has more content in it?

mgifford committed:
Update from Mike.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

mgifford commented on practice-areas/accessibility-how-to.md:
```diff
@@ -0,0 +1,37 @@
+# Accessibility: How To
+
+## Resources for learning how to test the accessibility of our applications
```
@maggiewachs what do you think of this extension?

mgifford replied:
@maggiewachs what do you think of this extension?

mgifford replied:
@maggiewachs what do you think of this extension?

maggiewachs commented on practice-areas/accessibility-how-to.md:
```diff
@@ -0,0 +1,37 @@
+
+Most operating systems have a screen reader built in. One of the most popular for blind users is JAWS, but the expense and customizability presents limitations for users. Most casual accessibility testers will leverage the built-in VoiceOver screen reader on a Mac and the open source [NVDA on Windows](https://www.nvaccess.org/download/) for most testing. If using NVDA, consider contributing from  ...
+
+It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn to empathize with blind users, as they are navigating through the page.
```
This is admittedly a nit-picky comment :) But I like to reframe empathy as needing to learn from screen reader users' expertise. I'd change this to: "It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA)."

mgifford replied:
That makes sense to me. Like that you include a link to a Smashing video. So just replace that sentence with line 25 or just part of it?

mgifford replied:
Ok, I've modified it. Now reads as:  
  
`It is very difficult for sighted users to understand how a blind person uses a screen reader. You have to essentially forget the visual layout and learn how to navigate the page content with headers, links and aria-regions. It is also important to observe and learn from the ways blind users navigate through the page. This video demonstrates [How A Screen Reader User Surfs The Web](https://www.youtube.com/watch?v=OUDV1gqs9GA).`

maggiewachs commented on practice-areas/accessibility-how-to.md:
```diff
@@ -0,0 +1,37 @@
+
+How to test: Apple has produced a great quantity of documentation on both the [MacOSX](https://support.apple.com/en-gb/guide/voiceover/welcome/mac) & [iOS](https://support.apple.com/en-gb/guide/iphone/iph3e2e415f/ios) versions of VoiceOver. WebAim also has useful resources for [MacOSX](https://webaim.org/articles/voiceover/) & [iOS](https://webaim.org/articles/voiceover/mobile) versions. Harvard  ...
+
+Helpful tips: Make sure you know how to turn it off before you turn it on.
```
Excellent point!

maggiewachs reviewed (commented):
This is great, ship it!

dmundra commented on practice-areas/accessibility-how-to.md:
```diff
@@ -0,0 +1,37 @@
+
+1/ [Accessibility Insights](https://accessibilityinsights.io/) (FastPass)
+
+Microsoft's open source Accessibility Insights offers both the automated testing with axe and guided keyboard only testing within their FastPass. This is roughly equivalent to using WebAim's proprietary [Wave](https://wave.webaim.org/) toolbar with \[Keyboard-only testing\](https://webaim.org/techniques/keyboard/). The main advantage of Accessibility Insights is that it provides assets which make ...
```
@mgifford looks like this link `\[Keyboard-only testing\](https://webaim.org/techniques/keyboard/)` is being escaped.

mgifford replied:
I think I squashed that now.

dmundra reviewed (changes requested):
@mgifford looking good, one minor link fix and then this can be merged.

mgifford committed:
No idea why the \'s were there.

hhartingrex committed:
Update accessibility-how-to.md

Added H3s. 
Note: I don't think I can add a folder.

hhartingrex committed:
Update accessibility-how-to.md

Made small type changes.

ContentBrewmaster reviewed (commented):
@dmundra @maggiewachs there seem to be unresolved comments in this content. Can you take a look?

dmundra committed:
Moved accessibility practice area documents to new folder. (#1092)

* Moved accessibility practice area documents to new folder. Updated links.

* Updated accessibility redirect.

dmundra reviewed (commented):
@hhartingrex, @mgifford FYI I merged my folder changes for the accessibility documents into this PR.

dmundra reviewed (approved):
Changes look good to me! @mgifford can you review the comment from Maggie?

mgifford committed:
Mod of empathy description with link to video

Pulling in suggestion from @maggiewachs https://github.com/CivicActions/guidebook/pull/1086#discussion_r1149304739
